docs/examples.md

```diff
@@ -164,6 +164,39 @@ jobs:
           comment-summary-in-pr: always
 ```
 
+## Getting the results of the action in a later step
+
+Using the `comment-content` output you can get the results of the action in a workflow step.
+
+```yaml
+name: 'Dependency Review'
+on: [pull_request]
+
+permissions:
+  contents: read
+  pull-requests: write
+
+jobs:
+  dependency-review:
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'Checkout Repository'
+        uses: actions/checkout@v4
+      - name: 'Dependency Review'
+        id: review
+        uses: actions/dependency-review-action@v4
+        with:
+          fail-on-severity: critical
+          deny-licenses: LGPL-2.0, BSD-2-Clause
+      - name: 'Report'
+        if: always()  # make sure this step runs even if the previous failed
```

Comment:
Had to add this to make sure this step was run regardless of the exit status of the previous one. When DR Action was run on a pull request and it failed it would stop execution (sample run here).

I have validated this workflow here: https://github.com/future-funk/bookish-eureka/pull/4. See the details for the `echo` output.

Reply:
@jsoref Gonna run with this example for the release, if you know of a better way to get around the Action failures please let me know and I'll be happy to update the docs.

Reply:
I think we want: [`jobs.<job_id>.steps[*].continue-on-error`](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepscontinue-on-error)

Reply:
From the description: `Set to true to allow a job to pass when this step fails`. We'd be marking the run as a success even if it fails, which is not a behavior we want. I'm not too familiar with workflows, so please let me know if I'm misreading this.

Reply:
If the goal is for the job to fail and to generally yield output, then you want: `${{ !cancelled() }}` see the note in https://docs.github.com/en/actions/learn-github-actions/expressions#always

Reply:
Thanks for the pointer. `!cancelled()` skipped the `Report` step: https://github.com/future-funk/bookish-eureka/actions/runs/7963028628/job/21737777423. 

There is a `if: ${{ failure() && steps.review.conclusion == 'failure' }}` condition in there that looks much more appropriate for this case. I was able to validate it's working in https://github.com/future-funk/bookish-eureka/actions/runs/7963058574/job/21737875771?pr=6. I'll update the example!

Reply:
Fixed in https://github.com/actions/dependency-review-action/commit/587ff57.